lib/mpl_toolkits/axes_grid1/axes_grid.py

```diff
@@ -38,21 +43,30 @@ def colorbar(self, mappable, *, ticks=None, **kwargs):
             if ticks is None:
                 ticks = ticker.MaxNLocator(5)  # For backcompat.
             from .colorbar import Colorbar
+            cb = Colorbar(
+                self, mappable, orientation=orientation, ticks=ticks, **kwargs)
+            self._cbid = mappable.callbacksSM.connect(
+                'changed', cb.update_normal)
```

Comment:
Mark as deprecated as well?

Reply:
Actually the whole block is deprecated since 3.2 (it's "legacy colorbar") so will be removed even before the deprecation on cbid expires.  The real point of the deprecation is to sync the "standard colorbar" case (which is below).